bookingticketui/src/pages/search-results-page/search-results.jsx

```diff
@@ -30,7 +30,7 @@ const ListSearch = () => {
   };
 
   const handleEconomyClick = async (event, id) => {
-    navigate(`/passenger-form/${id}`);
+    navigate(`/passenger-form/${id}/${"economy"}`);
     setFlight({
```

Comment:
${"economy"} ??

Reply:
i set this to take exactly  price for economy seat or business seat in flight info component